Multidimensional array in struct does not work with u32

**V version:** 
0.1.22 5536eb5

**OS:** 
Windows 10 (Linux WSL)

**What did you do?**
```v
struct Something {
  mut:
    arr [][]u32
}

fn main () {
    something := Something {
        arr : []
    }
}
```

**What did you expect to see?**
Compile without error
 
**What did you see instead?**
```
/tmp/v/test.tmp.c: In function ‘main__main’:
/tmp/v/test.tmp.c:3282:79: error: ‘array_u32’ undeclared (first use in this function); did you mean ‘array_f32’?
 Something something= (Something) { .arr =...
(Use `v -g` to print the entire error message)

V error: C error. This should never happen.
```

--- a/vlib/compiler/cheaders.v
+++ b/vlib/compiler/cheaders.v
@@ -194,6 +194,9 @@ typedef array array_int;
 typedef array array_byte;
 typedef array array_f32;
 typedef array array_f64;
+typedef array array_u16;
+typedef array array_u32;
+typedef array array_u64;
 typedef map map_int;
 typedef map map_string;
 #ifndef bool

